### ISSUE TITLE
DAILAController object has no attribute 'explian_decompilation'

### ISSUE THREAD

**Issue by larsw** [mahaloz/DAILA#8]
Exception when I try to "Explain function".

Looks like a typo?

```
[+] Starting DAILA Operations selector for xxxxxxxxxxxxx
Traceback (most recent call last):
  File "/home/larsw/ghidra_scripts/daila_ghidra.py", line 161, in <module>
    select_and_run_daila_op()
  File "/home/larsw/ghidra_scripts/daila_ghidra.py", line 156, in select_and_run_daila_op
    op_func()
  File "/home/larsw/ghidra_scripts/daila_ghidra.py", line 118, in explain_function
    resp = server.explain_function(decomp)
  File "/home/larsw/Downloads/ghidra_10.2_PUBLIC/Ghidra/Features/Python/data/jython-2.7.3/Lib/xmlrpclib.py", line 1243, in __call__
    return self.__send(self.__name, args)
  File "/home/larsw/Downloads/ghidra_10.2_PUBLIC/Ghidra/Features/Python/data/jython-2.7.3/Lib/xmlrpclib.py", line 1598, in _ServerProxy__request
    response = self.__transport.request(
  File "/home/larsw/Downloads/ghidra_10.2_PUBLIC/Ghidra/Features/Python/data/jython-2.7.3/Lib/xmlrpclib.py", line 1283, in request
    return self.single_request(host, handler, request_body, verbose)
  File "/home/larsw/Downloads/ghidra_10.2_PUBLIC/Ghidra/Features/Python/data/jython-2.7.3/Lib/xmlrpclib.py", line 1283, in request
    return self.single_request(host, handler, request_body, verbose)
  File "/home/larsw/Downloads/ghidra_10.2_PUBLIC/Ghidra/Features/Python/data/jython-2.7.3/Lib/xmlrpclib.py", line 1316, in single_request
    return self.parse_response(response)
  File "/home/larsw/Downloads/ghidra_10.2_PUBLIC/Ghidra/Features/Python/data/jython-2.7.3/Lib/xmlrpclib.py", line 1493, in parse_response
    return u.close()
  File "/home/larsw/Downloads/ghidra_10.2_PUBLIC/Ghidra/Features/Python/data/jython-2.7.3/Lib/xmlrpclib.py", line 800, in close
    raise Fault(**self._stack[0])
xmlrpclib.Fault: <Fault 1: "<class 'BaseException'>:'DAILAController' object has no attribute 'explian_decompilation'">
daila_ghidra.py> Finished!
```

**Comment by mahaloz** [comment]
@larsw good catch. Looks like a merge failed in some history of the commit log. Try now.

**Comment by mahaloz** [comment]
Closing since I've fixed it in #9
